src/networking.c

```diff
@@ -209,6 +210,91 @@ cc_oci_tap_create(const gchar *const tap) {
 }
 
 /*!
+ * Helper function for setting/getting MTU for network interface.
+ *
+ * \param ifname Interface name.
+ * \param mtu MTU to be set/retrieved for the interface.
+ * \param set Flag indicating if the mtu needs to be set/retrieved.
+ *
+ * \return \c true on success, else \c false.
+ */
+gboolean
+cc_oci_handle_interface_mtu(char *ifname, unsigned int *mtu, gboolean set)
```

Comment:
I was just thinking that. I was contemplating just how hard it would be to pass an invalid pointer to it. So, if we are being picky ;-) (and we usually are....), I'll add another comment....